.github/workflows/nightly.yml

```diff
@@ -35,7 +35,6 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
       - name: Release nightly
         uses: softprops/action-gh-release@v1
-        if: github.ref == 'refs/heads/main'
```

Comment:
can we do a check to see if changes are in certain files (`*.rs`) because is overwhelming receive tags for every PR merge.

Reply:
maybe something like this https://github.com/dorny/paths-filter ?

Reply:
Or https://github.com/tj-actions/changed-files

Reply:
cc @devhector (if you want to take a look as well)

Reply:
That would be for another issue, not this contribution

Reply:
Agree, approving.